content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -42,12 +40,12 @@ Accounting entries will depend on the accounting mode: *Continental* or *Anglo-S
 
```

Comment:
I can't make a suggestion to a grey part of this doc, but right above here in the tip block, it says ot chec what ind of accounting it is- but I do not see where to see that? Is it ONLY verified in the "Stock Valuation" section as ecplained below? If so, I would make that more clear- I was tyring to find out where to verify the info.

Reply:
Thanks for pointing this out! Added more info here (: 

Appreciate your help in reading through the doc (even the gray parts!) and making sure all instructions are helpful and comprehensive :blush:

Reply:
Comment seems settled by what's written in the **tip** block, can resolve.